witness/src/Dscp/Witness/CLI.hs

```diff
@@ -96,6 +97,15 @@ witnessKeyParamsParser = do
     wkpCommittee <- optional committeeParamsParser
     pure $ WitnessKeyParams {..}
 
+feeCoefficientsParser :: Parser FeeCoefficients
```

Comment:
Noooooo, it should be in configuration. Configuration is a set of "cluster-related" values -- it doesn't make sense to change this value for a single cluster user to anything else -- you'll either pay more or your tx will be rejected, or you'll create an incorrect block.

PS: core configuration. Wallet (that doesn't want to know anything about witness' code) should be able to create a proper tx.